src/core/Microsoft.PowerToys.Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -146,7 +146,19 @@
 	                IsChecked="{x:Bind Mode=TwoWay, Path=ViewModel.DisableDriveDetectionWarning}"
 	                IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.EnablePowerLauncher}"
 	                />
+            <TextBlock x:Uid="PowerLauncher_Theme"
+                       Style="{StaticResource SettingsGroupTitleStyle}"
+                       Foreground="{x:Bind Mode=OneWay, Path=ViewModel.EnablePowerLauncher, Converter={StaticResource ModuleEnabledToForegroundConverter}}" />
+            <StackPanel AutomationProperties.LabeledBy="{Binding ElementName=RadioButtons_Name_Theme}">

```

Comment:
Add a TextBlock here that is the title textblock for the radiobuttons (make sure to not use the SettingsGroupTitleStyle). Add:

            <TextBlock
               x:Uid="PowerLauncher_Theme"
                Foreground="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled, Converter={StaticResource ModuleEnabledToForegroundConverter}}"/>

Reply:
But then we don't have an additional settings group. And I think we should not add the colour settings to the group "search and results".